src/commands/tsgen.ts

```diff
@@ -45,6 +45,14 @@ export default class TypeScriptCodeGeneratorCommand extends Command {
       default: true,
       allowNo: true,
     }),
+
+    branch: flags.string({
+      char: 'b',
+      description: 'branch',
+      hidden: false,
+      multiple: false,
+      required: true,
```

Comment:
@vkalta Can you make this branch flag optional? We have some customers who do not have branches enabled yet.